lib/chef_zero/server.rb

```diff
@@ -98,7 +100,6 @@ class Server
 
     GLOBAL_ENDPOINTS = [
       '/license',
-      '/users',
```

Comment:
It likely doesn't matter wrt chef-client currently because of our poor users support.  But users is both a globa and per-org endpoint.